regression-test/suites/trino_p0/constant_group_key.groovy

```diff
@@ -0,0 +1,35 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// This suit test the `backends` information_schema table
+suite("constant_group_key") {
+    explain {
+        sql("select 'oneline' from nation group by 'constant1'")
+        contains "group by: 'constant1'"
+    }
+
+    explain {
+        sql("select 'oneline' from nation group by 'constant1', 'constant2'")
+        contains "group by: 'constant1'"
+    }
+
+    //no group by key
+    explain {
+        sql("select 'oneline', count(*) from nation group by 'constant1', 'constant2'")
+        contains "group by: \n"
```

Comment:
scalar agg is not same with vector agg, so we could not remove all constant group by exprs.